README.rst

```diff
@@ -42,14 +42,32 @@ path to the SQLite database.
 
     $ xbrl_extract {path_to_filings} {path_to_database}
 
+This repo contains a small selection of FERC Form 1 filings from 2021 in the
+``examples`` directory. To test the tool on these filings, use the command
+
+.. code-block:: console
+
+    $ xbrl_extract examples/ferc1-2021-sample.zip {path_to_database}
+
 By default, the CLI will use the 2022 version of the FERC Form 1 Taxonomy to create
```

Comment:
Is it specifically using the **2022** taxonomy by default? Or the **current year**? If it's specifically 2022, I imagine we would want to update it each year.

Reply:
Right now it is specifically the 2022 version. I'm not sure when exactly they will release new versions, so I didn't want to try to always use the current year